File: birdeye.py
from urllib.request import Request, urlopen
import xmltodict
import json

request = Request(
    'https://api.birdeye.com/resources/v1/review/businessid/158170949468469/summary?api_key=bJQGlIUGx2ZZ9YlnuVPPYMVR3yhrmWAU')

response_body = urlopen(request).read()
# print(response_body)
my_dict = xmltodict.parse(response_body)
json_data = json.dumps(my_dict)

print(json_data)

File: birdeye.js
var request = new XMLHttpRequest();

request.open('GET', 'https://api.birdeye.com/resources/v1/review/businessid/158170949468469/summary?api_key=bJQGlIUGx2ZZ9YlnuVPPYMVR3yhrmWAU');

request.onreadystatechange = function() {
    if (this.readyState === 4) {
        console.log('Status:', this.status);
        console.log('Headers:', this.getAllResponseHeaders());
        console.log('Body:', this.responseText);
    }
};

request.send();